{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/core/views.py

```diff
@@ -50,45 +50,45 @@ def post(self, request, *args, **kwargs):
         return Response(response_data)
 
 
-class UserViewSet(
-    viewsets.GenericViewSet,
-    mixins.RetrieveModelMixin,
-    mixins.ListModelMixin,
-    mixins.UpdateModelMixin,
-    mixins.DestroyModelMixin,
-):
-    queryset = User.objects.all()
+class UserViewSet(viewsets.GenericViewSet, mixins.RetrieveModelMixin):
     serializer_class = UserSerializer
 
     # No auth required to create user
     # Auth required for all other actions
-    permission_classes = (permissions.IsAuthenticated | CreateOnlyPermissions,)
+    permission_classes = (HasUserPermissions,)
 
-    @transaction.atomic
-    def create(self, request, *args, **kwargs):
+    def get_queryset(self):
         """
-        Endpoint to create/register a new user.
+        Users should only find themselves by default
         """
+        user = self.request.user
+        return User.objects.filter(pk=user.pk)
```

Comment:
@whusterj I removed the querset and replaced it with this call. It might not matter since we don't have list as an option here anymore. But probably a good default to have for projects that try adding a quick mixin here.

Reply:
aaaand it's broken the tests. Will investigate in the morning.

Reply:
I had to add the queryset back. Left a comment as to why